breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py

```diff
@@ -0,0 +1,31 @@
+from breathecode.events.models import Event
+from breathecode.events.serializers import EventSerializer
+from breathecode.utils import ScriptNotification
+from django.utils import timezone
+from breathecode.utils.datetime_interger import duration_to_str, from_now
+
+published_without_live_stream_url = Event.objects.filter(status='ACTIVE',
+                                                         online_event=True,
+                                                         live_stream_url='',
+                                                         ending_at__gt=timezone.now())
+total = published_without_live_stream_url.count()
+
+if total > 0:
+    msg = ''
+    for event in published_without_live_stream_url:
+        msg += f'- <a href="{ADMIN_URL}/events/event/{event.id}">{event.title}</a> added {from_now(event.created_at)} ago. \n'
+
+        event.status = 'draft'
+        serializer = EventSerializer(data=event, context={'academy_id': None})
```

Comment:
Is not necessary use the serializer, you can use `event.save()` or `published_without_live_stream_url.update(status='DRAFT')`